eagle-xl/registration.py

```diff
@@ -164,3 +166,23 @@
         ),
     )
 
+# Now baryon fractions
+
+Omega_m = catalogue.units.cosmology.Om0
+Omega_b = catalogue.units.cosmology.Ob0
+
+M_500 = catalogue.spherical_overdensities.mass_500_rhocrit
```

Comment:
This needs to be protected in a similar way as the HI/H2 above.

Reply:
Ok, but we don't have a reasonable replacement here. Do you want me to just propose an array of zeros?

Reply:
I think an array of ones here would be best as that wouldn't lead to crashes. Let's do that for now and I'll revisit this once I have actually got the 'only show this plot if you could make it' working properly.

Reply:
Done. :)